lib/react-typescript.js

```diff
@@ -2,5 +2,7 @@ module.exports = {
   rules: {
     "react/jsx-filename-extension": ["error", { extensions: [".tsx"] }],
     "react/prop-types": ["off", {}],
+    "no-shadow": "off",
+    "@typescript-eslint/no-shadow": "error",
```

Comment:
Could you remove this change in this file?
This file should only include rules for React & TypeScript and the `no-shadow` rule is related to TypeScript.
We expect `lib/react-typescript` is used with `lib/typescript`, so we don't have to have this setting besides in `lib/typescript`.

We expect that `lib/*` is an internal library and `presets/*` is a public interface, so we don't intend `lib/*` is used directly in ESLint configs.

Reply:
Thanks for the review!
I am sorry for the lack of understanding. I understand the difference between `lib/*` and `presets/*`. I removed the changes in the file, and could you review it again please?